tests/e2e/identity-and-authorization/create-with-email.test.ts

```diff
@@ -0,0 +1,48 @@
+import {NightWatchBrowser} from 'nightwatch';
+
+import * as admin from 'firebase-admin';
+
+const {
+        FIREBASE_DATABASE_URL,
+        EMAIL_AND_PASSWORD_TEST_EMAIL,
+      } = process.env;
+
+function getAuthAdmin() {
+  const serviceAccount = require('../../../../firebase.json');
```

Comment:
You cannot require a local JSON file. It doesn’t exist in the repository and neither should because of security reasons. These tests will fail on CI. As, hopefully, clearly described in the README, the values of the JSON export must be stored in environment variables locally, so we can have consistent testing that will also pass on CI.